### ISSUE TITLE
Suggestion - Changing police 10-14 alarm and allowing it to be heard by players around the officer.

### ISSUE THREAD

**Issue by ItzBrittanyBitxh** [InZidiuZ/op-framework-issue-tracker#3110]
### Assurance

I have read the `README.md`

### Summary

Change the sound of the 10-14 tracker alarm when it's pressed to make it more impactful and... Scary so its recognized more. Also make it so the sound effect is heard by anyone around the vicinity of the officer at the time they are downed, not just over the radio.

### Reason

 A lot of the time, when an officer does down, its easily looked over and sometimes not taken seriously, if the tracker was changed to something a little scarier.. so to speak then it may be noticed more. Additionally, it would be good for players around to hear if the tracker has been used or not. 

Imagine if a crim has taken an officer hostage and downed them for example, it sets off a type of fear to physically hear and know that the tracker went off to all of pd. 

A new sound similar to one like this: https://www.youtube.com/watch?v=KsS7IZ0GBMM  would be cool like a 2-3 second sound which is also still played over radio to other officers.

**Comment by scufflord** [comment]
+1 on this, it just adds a lot more to a scene imo

**Comment by coalaura** [comment]
> A new sound similar to one like this: https://www.youtube.com/watch?v=KsS7IZ0GBMM

This would get incredibly annoying over radio. Idm making it play in the area

**Comment by ItzBrittanyBitxh** [comment]
Yeah maybe not that one exactly, just a sound that plays for a second longer then the current one, as its very easy to miss if you are not paying attention.

**Comment by scufflord** [comment]
What about the classic SOS morse code beeps? https://www.youtube.com/watch?v=N4P05TASXMs (obviously not 11 seconds of it but this is just an example)

**Comment by coalaura** [comment]
> What about the classic SOS morse code beeps? https://www.youtube.com/watch?v=N4P05TASXMs (obviously not 11 seconds of it but this is just an example)

also pretty annoying imo

**Comment by MetaHatter** [comment]
Wilhelm Scream?
https://www.youtube.com/watch?v=r6JK-gRELI0

**Comment by coalaura** [comment]
> Wilhelm Scream? https://www.youtube.com/watch?v=r6JK-gRELI0

a lot less annoying on the radio LMAOOO, reminds me of @ImArteria s version

**Comment by ItzBrittanyBitxh** [comment]
Yeah DEFS not lol. Any kind of short alarm would be good

**Comment by ImArteria** [comment]
> > Wilhelm Scream? https://www.youtube.com/watch?v=r6JK-gRELI0
> 
> a lot less annoying on the radio LMAOOO, reminds me of @ImArteria s version

I thought about responding with the link to it but yk. yk.
